rbx_reflection/src/class_tag.rs

```diff
@@ -18,6 +18,7 @@ pub enum ClassTag {
 }
 
 #[derive(Debug)]
+#[allow(dead_code)]
```

Comment:
Why this?

Reply:
In Rust 1.77.0, the `unused_tuple_struct_fields` lint was merged into the `dead_code` lint: https://github.com/rust-lang/rust/pull/118297/. This caused our lint check to fail, but I think we should keep the field.

Reply:
Fair enough. I guess that makes sense.